tools/nvqpp/nvq++.in

```diff
@@ -443,6 +473,10 @@ while [ $# -ne 0 ]; do
 		OUTPUTFILE="$2"
 		shift
 		;;
+	-o=*)
```

Comment:
I'm not sure I like this addition in particular. Do we really want `-o=<filename>`? Or even generally the `=` syntax for every switch that takes an argument? It seems like this may be creeping complexity; it's not clear there is a need or advantage to some of these.

Reply:
I agree, I will discuss and have a follow-up PR to address these points.